node/src/accountant/mod.rs

```diff
@@ -387,6 +494,42 @@ impl Accountant {
             .expect("BlockchainBridge is dead");
     }
 
+    fn scan_for_pending_payments(&self) {
+        debug!(self.logger, "Scanning for pending payments");
+        let filtered_pending_payments = self.pending_payments_dao.return_all_payment_backups();
+        if !filtered_pending_payments.is_empty() {
```

Comment:
This would look a little better if the clauses were reversed, so that we could do xxx.is_empty() rather than !xxx.is_empty(), but that's a nitpick.